src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -908,7 +921,7 @@ static inline int MPIDI_OFI_do_accumulate(const void *origin_addr,
         goto am_fallback;
 
     MPIDI_CH4U_RMA_OP_CHECK_SYNC(target_rank, win);
-    if (target_rank == MPI_PROC_NULL)
+    if (unlikely(target_rank == MPI_PROC_NULL))
```

Comment:
Looks like this was added in all places.  Is this really a consistency change?

Reply:
Yes. Originally only some of the places have `unlikely`, such as those in `MPIDI_NM_mpi_put`, `MPIDI_NM_mpi_get`, `MPIDI_NM_mpi_compare_and_swap`.